linode_api4/objects/placement.py

```diff
@@ -0,0 +1,100 @@
+from dataclasses import dataclass
+from enum import StrEnum
+from typing import List, Union
+
+from linode_api4.objects.base import Base, Property
+from linode_api4.objects.linode import Instance
+from linode_api4.objects.region import Region
+from linode_api4.objects.serializable import JSONObject
+
+
+class PlacementGroupAffinityType(StrEnum):
+    """
+    An enum class that represents the available affinity policies for Linodes
+    in a Placement Group.
+    """
+
+    anti_affinity_local = "anti_affinity:local"
+
+
+@dataclass
+class PlacementGroupMember(JSONObject):
```

Comment:
In the future it'd be nice to have a property method to access the underlying Linode object from here.